src/chainstate/burn/db/burndb.rs

```diff
@@ -319,6 +332,8 @@ const BURNDB_SETUP : &'static [&'static str]= &[
     CREATE TABLE snapshots(
         block_height INTEGER NOT NULL,
         burn_header_hash TEXT UNIQUE NOT NULL,
+        sortition_id TEXT UNIQUE NOT NULL,
+        pox_id TEXT NOT NULL,
```

Comment:
Do we expect `pox_id` to get progressively longer as more and more reward windows are processed?  If so, should this just be a foreign-key `rowid` to a table of `pox_ids` so that we only ever store the `pox_id` once, anywhere?

Reply:
No -- I don't think so, I think the `pox_id` will ultimately be something more like what the MARF fork table used to maintain -- a unique "fork identifier" that would be able to disambiguate between sortitions in the same burnchain block, but would require asking the pox_db to determine other kinds of properties about the pox_id (e.g., answering whether a pox_id is a child fork of a parent pox_id).

Reply:
Ah, I see below that `PoxIdentifier` is a 32-byte value, and that's what's getting stored here.  I think I may be confusing this with the proposed "PoX lock log," which is a bitvector of all reward cycles that had an anchor block.  Is this stored anywhere?

Reply:
Though that does ultimately mean it would end up being a foreign key to something like a rowid (the pox_id itself would be the rowid). I'll make a note in https://github.com/blockstack/stacks-blockchain/issues/1551 to make sure to revisit adding a foreign key relationship here when implementing the PoxDB.

Reply:
> "PoX lock log," which is a bitvector of all reward cycles that had an anchor block. Is this stored anywhere?

That'll depend on the PoxDB implementation ultimately -- I think the "PoX bitvector" is one approach to implementing that, but we'll see what turns out to be the best. Right now, it's just be treated as an opaque identifier (and the PoxDB has to be consulted to determine things like parent/child identifier relationships).